ako-clean/cleanup.go

```diff
@@ -406,6 +408,21 @@ func cleanupPools() error {
 	return deleteAviResource("/api/pool", pools)
 }
 
+func cleanupAppProfiles() error {
+	tenant := lib.GetAdminTenant()
+	aviClient := avicache.SharedAVIClients(tenant).AviClient[0]
+	err, appProfs := lib.ProxyEnabledAppProfileGet(aviClient)
+	if err != nil {
+		utils.AviLog.Warnf("Application profile Get returned err %v", err)
+		return err
```

Comment:
are we checking response status code, we should not return error in case of 404

Reply:
this is api query "?name=<value>" , will not return 404 but blank result list.